RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,118 @@
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal overview
+
+I believe we should rethink the licensing terms for website content, as listed on [thegooddocsproject.dev/licenses/](https://thegooddocsproject.dev/licenses/). As they're currently written, they provision everything as [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (a.ka. "CC BY 4.0), including all text, images, HTML code, and anything else.
+
+We may not always have the rights to relicense what we put on our page, and not every contributor is comfortable with having their info and likeness released as publicly usable.
+
+### Scope
+
+The scope of this is about the website's content. Licensing for our products (templates, doc tools, etc.) aren't part of this discussion, though this proposal does indicate where the primary home for those licenses should be.
+
+## Motivation
+
+For the upcoming blog, not everything we release will be ours. If we use stock photos for the purposes of social media optimization, we won't have the rights to relicense that content. We also may not have the rights to relicense other content we'll reuse, including any HTML/CSS/scripting to generate the pages.
+
+On a personal level, I don't release my likeness to public use, and the current website license terms makes me wary of continuing with the project. I used to work in games (and still do at times), and I've had too many death threats (honestly, one such threat is too many) and have seen too many friends be impersonated to want to give bad actions a contractual right to use my likeness however they like. (Bad actors will still use likenesses, but at least they can't defend their actions by citing a license that allows them to use one's info.)
+
+### Blocking
+
+[RFC002 (Procope/blog & video)](https://github.com/thegooddocsproject/request-for-comment/pull/3) is blocked by this proposal.
+
+## Proposal
+
+I'd like to shift the burden of licensing to the individual elements, rather than have a pure blanket. That means:
+
+- Our products (templates, doc tools, etc) have the BSD0 license in their readme or license text files, and not rely on the license website page for that notice
+- For blog posts etc, post pages would default to have "`This blog post's original text is (c) [year] [author name], and licensed under CC BY 4.0`". Currently, the blog is set up to allow for this default, with other changes allowed. Ryan and Bryan worked together to make sure this could be modular.
+- "Original text" covers not opening any quoted text (like in referencing a book's text) as being CC, and doesn't open any stock images from being CC. If a custom image is created by the author, we can add that to the CC license, which is part of why Ryan and Bryan worked to make the license field modular.
+- For everything else, we leave the material either unlicensed or under whatever licenses already exist for that material. (See What's currently CC BY content below.)
```

Comment:
suggestion (blocking): I'm concerned about us supporting unlicensed or non-open licensed material. This makes it hard for downstream users to re-purpose content on our site (which is counter to the principles of open source). We should make it easy for downstream users to build upon ideas on our site, which we do by making our licenses open, clear, and simple to use.
I expect the majority of our contributors will support this.
The deal you make with open source is:
* You get the benefits of open source created before you.
* You get the marking pipeline provided by a successful open source project.
* Your content can then be used for free by others, which reduces any proprietary lock in you can provide (to say write a book and prevent others from using your ideas.)

Reply:
I don't see how this isn't handling that. Admittedly, I also don't see why we need CC BY for this content except to allow those of us at Google to freely submit to this. Using the Write the Docs model, the information is freely released without a CC BY model, because the intent of the information is to be repurposes through thought, not copy-paste. Thus, downstream users are able to bulid upon the ideas on our site.

But since contributors are releasing the info as CC BY, we're being even more permissive than that--not less--by allowing literal replication.

Reply:
I'm trying to be forward-thinking here by allowing us to quote some non-open text to build on it. Or to even quote text in general without having to research the potential licensing just to make a blog post framed around building on communal knowledge—like responding/adding to something mentioned in a WTD talk.

Reply:
Comment (agreeing with Ryan): I also don't understand the nature of the objection you're making Cameron. The ideas are publicly accessible and quotable. People can build on those ideas and remix them to make even better ideas. CC BY is a pretty open license type. The only requirement is that people need to give proper attribution to a blog when it informs their thinking, when they quote it, or when they are inspired by it. I think that's a generally ethical practice anyhow: give credit to the people who came up with a good idea that you like.

Reply:
Thanks for the considered response. I'll recategorize my comment to:
thought (non-blocking): I'm considering we might want to use:
We leave the material under whatever licenses already exists, or if unspecified, it we default to the CC-By.

My thinking is derived from what I suggest are norms. Refer to github terms: https://docs.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license

### 6. Contributions Under Repository License
Whenever you add Content to a repository containing notice of a license, you license that Content under the same terms, and you agree that you have the right to license that Content under those terms. If you have a separate agreement to license that Content under different terms, such as a contributor license agreement, that agreement will supersede.

Isn't this just how it works already? Yep. This is widely accepted as the norm in the open-source community; it's commonly referred to by the shorthand "inbound=outbound". We're just making it explicit.

Reply:
If unspecified, we'll leave unspecified. If someone didn't specify it and we're still using it for some reason, we are not going to decide for them it's CC BY. Deciding the license terms for someone else's content is a hard blocker.

Reply:
Added the consolidated version of this to feedback section for posterity